airflow/contrib/example_dags/example_gcp_automl_operators.py

```diff
@@ -0,0 +1,306 @@
+        dataset=DATASET,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+
+    dataset_id = (
+        '{{ task_instance.xcom_pull("create_dataset_task", key="dataset_id") }}'
+    )
+
+    import_dataset_task = AutoMLImportDataOperator(
+        task_id="import_dataset_task",
+        dataset_id=dataset_id,
+        location=GCP_AUTOML_LOCATION,
+        input_config=IMPORT_INPUT_CONFIG,
+    )
+
+    list_tables_spec_task = AutoMLTablesListTableSpecsOperator(
+        task_id="list_tables_spec_task",
+        dataset_id=dataset_id,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+
+    list_columns_spec_task = AutoMLTablesListColumnSpecsOperator(
+        task_id="list_columns_spec_task",
+        dataset_id=dataset_id,
+        table_spec="{{ get_object_id(task_instance.xcom_pull('list_tables_spec_task')[0]) }}",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+
+    # [START howto_operator_list_dataset]
+    list_datasets_task = AutoMLDatasetListOperator(
+        task_id="list_datasets_task",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_list_dataset]
+
+    # [START howto_operator_delete_dataset]
+    delete_datasets_task = AutoMLDeleteDatasetOperator(
+        task_id="delete_datasets_task",
+        dataset_id="{{ task_instance.xcom_pull('list_datasets_task', key='dataset_id_list') | list }}",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_delete_dataset]
+
+    (
+        create_dataset_task  # noqa
+        >> import_dataset_task  # noqa
+        >> list_tables_spec_task  # noqa
+        >> list_columns_spec_task  # noqa
+        >> list_datasets_task  # noqa
+        >> delete_datasets_task  # noqa
+    )
+
+with models.DAG(
+    "example_gcp_get_deploy",
+    default_args=default_args,
+    schedule_interval=None,  # Override to match your needs
+) as get_deploy_dag:
+    # [START howto_operator_get_model]
+    get_model_task = AutoMLGetModelOperator(
+        task_id="get_model_task",
+        model_id=MODEL_ID,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_get_model]
+
+    # [START howto_operator_deploy_model]
+    deploy_model_task = AutoMLDeployModelOperator(
+        task_id="deploy_model_task",
+        model_id=MODEL_ID,
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_deploy_model]
+
+
+with models.DAG(
+    "example_gcp_predict",
+    default_args=default_args,
+    schedule_interval=None,  # Override to match your needs
+) as predict_dag:
+    # [START howto_operator_prediction]
+    predict_task = AutoMLPredictOperator(
+        task_id="predict_task",
+        model_id=MODEL_ID,
+        payload={},  # TODO Add your own payload, the used model_id must be deployed
```

Comment:
No, `model_id` is not a part of payload. The payload is described in AutoML docs and it depends on service type. I assume that user will be familiar / will check the official REST API / Python SDK. I think it could be worth to put there a comment with link to payload definition. @lwander WDYT?